gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java

```diff
@@ -54,6 +56,9 @@
 
 	private final int NULL_SENTINEL = 127;
 
+	private static final String SAK_PROP_SHOW_SET_ZERO_SCORE = "gradebookng.showSetZeroScore";
+	private static final boolean SAK_PROP_SHOW_SET_ZERO_SCORE_DEFAULT = true;
+
```

Comment:
I wouldn't bother with either of these, tbh. You're only using them once so it helps if the string is defined as and when it is used, for readability's sake. That's my take anyway.

Reply:
Yeah, it's just my personal preference, but I see your point. Ideally, I think there should be a common constants class where all sakai.properties and their defaults are defined once, and can be referenced from any project.